### PR TITLE
refactor:KeyLabler

### PR DESCRIPTION
**Pull request by EncryptedDev**
Removed the `keylabeler.yml` file for the following reasons:
- We don't use the `[Presence | Bug]` naming convention for issues as we use labels now
- We do not use the `[Presence | Feature Request]` naming convention as we use discussions now

### CODE REVIEW

**Comment by ImRodry** [comment]
Not entirely within the scope of this PR but you should add a link in the ISSUE_TEMPLATE/config.yml file that sends users to discussions if they wanna make a feature request. Would make it more obvious imo

**Comment by EncryptedDev** [comment]
> Not entirely within the scope of this PR but you should add a link in the ISSUE_TEMPLATE/config.yml file that sends users to discussions if they wanna make a feature request. Would make it more obvious imo

We have the pinned issue to do so & we can convert issues to discussions with a click of a button so it isn’t really needed

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 blank_issues_enabled: false
 contact_links:
-  - name: Official Discord server
+  - name: Discord Support
```
Comment (ImRodry) [inline .github/ISSUE_TEMPLATE/config.yml:3]: I think the old name was just fine tbh

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
     url: https://discord.premid.app
-    about: Please use our ticket system for urgent requests!
+    about: If you are experiencing issues with your PreMiD, go to our Discord server for help.
```
Comment (ImRodry) [inline .github/ISSUE_TEMPLATE/config.yml:5]: Should still mention the ticket system, unless you want people to just message in a random chat which is also fine

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -1,5 +1,8 @@
...
+  - name: Feature Requests & Presence Requests
+    url: https://github.com/PreMiD/Presences/discussions
```
Comment (ImRodry) [inline .github/ISSUE_TEMPLATE/config.yml:7]:
```suggestion
    url: https://github.com/PreMiD/Presences/discussions/new
```

**Review by ImRodry — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 3 above)
Comment (EncryptedDev) [reply .github/ISSUE_TEMPLATE/config.yml:7]: Nope, as they need to copy the template before opening the discussion

Comment (ImRodry) [reply .github/ISSUE_TEMPLATE/config.yml:7]: Ah right, good luck with getting people to notice that
Might as well just link that discussion and have a link to the discussion creation page over there

**Pull request merged**